pages/reference/OS/configuration.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Configuration
+excerpt: Instructions for editing and customizing config.json for modifying the behavior of {{ $names.os.lower }}
+---
+
+# Configuring {{ $names.os.lower }}
+
+The behavior of {{ $names.os.lower }} can be configured by editing the `config.json` file. This file is located in the [boot partition][boot-partition] and accepts a [range of fields](#valid-fields) to modify the behavior of the host OS.
+
+**Note:** The `config.json` file is different from the [`config.txt`][config-txt] file, also located in the boot partition, which is used by the Raspberry Pi to set device configuration options.
+
+Before the device is [provisioned][provisioned], you may edit `config.json` by mounting a flashed SD card (which will mount as `resin-boot`) and editing the file directly. The boot partition is mounted on the device at `/mnt/boot`, and so on the device, the file is located at `/mnt/boot/config.json`. For example, to output a formatted version of `config.json` on a device, use the following commands:
```

Comment:
I wonder if "(with the partition-label `resin-boot`)" is better than "(which will mount as `resin-boot`)"; since I guess the latter is OS-specific? (e.g. think of Windows users)  :man_shrugging: